### DISCUSSION TITLE
should i go sideway and learn VRF v2 or stick to v1?

### DISCUSSION THREAD

**Discussion by ofirbenatar** [smartcontractkit/full-blockchain-solidity-course-py#1171] [Q&A] [answered]
my background: i been learning programming for the past 4 months (python + littilebit C).
should i implement VRF v2 for the course or go with V1 for now and learn it later?

**Comment by cromewar** [comment, marked as answer]
Hello @ofirbenatar 
For learning purposes I'll suggest to keep on v1 and later try to implement v2 with a deep knowledge.
